Mods/SML/Source/SML/Private/Toolkit/KismetBytecodeDisassembler.cpp

```diff
@@ -1050,16 +1053,19 @@ TSharedPtr<FJsonObject> FSMLKismetBytecodeDisassembler::SerializeExpression(int3
 		}
 	default:
 		{
-			// This should never occur.
-			checkf(0, TEXT("Unknown bytecode 0x%02X"), (uint8) Opcode);
+			// This should only really occur if the caller has requested an unaligned index.
+			fgcheckf(0, TEXT("Unknown bytecode 0x%02X at ScriptIndex %d. The supplied index is not aligned with a valid instruction!"), (uint8)Opcode, ScriptIndex);
```

Comment:
What does alignment have to do with an opcode not being known?

Reply:
AFAIK, there are two cases where this can happen:
1) A new opcode has been added to the instruction set and needs to be handled
2) The index supplied to SerializeExpression is not actually the position of an opcode.  I ran into this a few times while testing my code and finding the places I hadn't updated indexes correctly.

I updated the comment and error message a bit - let me know that helps clarify.

Reply:
It does help: the message before made it seem as if a bad index was the only reason for the assertion to fail.

Ack